manifests/user.pp

```diff
@@ -114,4 +127,8 @@
     }
   }
 
+  anchor { 'jboss::user::before':
```

Comment:
!fix You couldn't use constant name of the resource in defines. Resource duplication error when managing 2 or more users